multicluster/config/multi-cluster.yaml

```diff
@@ -1437,91 +998,62 @@ spec:
                         type: array
                       ports:
                         items:
-                          description: ServicePort represents the port on which the
-                            service is exposed
+                          description: ServicePort represents the port on which the service is exposed
                           properties:
                             appProtocol:
-                              description: The application protocol for this port.
-                                This field follows standard Kubernetes label syntax.
-                                Un-prefixed names are reserved for IANA standard service
-                                names (as per RFC-6335 and http://www.iana.org/assignments/service-names).
-                                Non-standard protocols should use prefixed names such
-                                as mycompany.com/my-custom-protocol. Field can be
-                                enabled with ServiceAppProtocol feature gate.
+                              description: The application protocol for this port. This field follows standard Kubernetes label syntax. Un-prefixed names are reserved for IANA standard service names (as per RFC-6335 and http://www.iana.org/assignments/service-names). Non-standard protocols should use prefixed names such as mycompany.com/my-custom-protocol. Field can be enabled with ServiceAppProtocol feature gate.
                               type: string
                             name:
-                              description: The name of this port within the service.
-                                This must be a DNS_LABEL. All ports within a ServiceSpec
-                                must have unique names. When considering the endpoints
-                                for a Service, this must match the 'name' field in
-                                the EndpointPort. Optional if only one ServicePort
-                                is defined on this service.
+                              description: The name of this port within the service. This must be a DNS_LABEL. All ports within a ServiceSpec must have unique names. When considering the endpoints for a Service, this must match the 'name' field in the EndpointPort. Optional if only one ServicePort is defined on this service.
                               type: string
                             port:
                               description: The port that will be exposed by this service.
                               format: int32
                               type: integer
                             protocol:
                               default: TCP
-                              description: The IP protocol for this port. Supports
-                                "TCP", "UDP", and "SCTP". Default is TCP.
+                              description: The IP protocol for this port. Supports "TCP", "UDP", and "SCTP". Default is TCP.
```

Comment:
some of these changes look unrelated; probably because we didn't run make manifest in the past?

Reply:
yes, I think I forgot to run make manifest last time. new labels are added now. the line format are new but it's controlled by generator, no change introduced in configuration.